haiku/_src/initializers.py

```diff
@@ -254,3 +254,42 @@ def __call__(self, shape: Shape, dtype: DType) -> jnp.ndarray:
     q_mat = jnp.reshape(q_mat, (n_rows,) + tuple(np.delete(shape, self.axis)))
     q_mat = jnp.moveaxis(q_mat, 0, self.axis)
     return jax.lax.convert_element_type(self.scale, dtype) * q_mat
+
+
+  class Zeros(Initializer):
+    """Initializer that generates tensors initialized to 0."""
+
+    def __call__(self, shape: Shape, dtype: DType) -> jnp.ndarray:
+      return jnp.zeros(shape, dtype)
+
+  class Ones(Initializer):
+    """Initializer that generates tensors initialized to 1."""
+
+    def __call__(self, shape: Shape, dtype: DType) -> jnp.ndarray:
+      return jnp.ones(shape, dtype)
```

Comment:
I think for Zeros and Ones it is probably more readable to just use `jnp.zeros` and `jnp.ones`. These actually match the `Initializer` signature so type checking works too 😄

Reply:
(In Sonnet we used a type for `Initializer` while in Haiku we just used `Callable[.., ..]` so any callable with the right signature is a valid init function.)

Reply:
So I remove them?

Reply:
Thanks, looks good now.